torch_xla/core/xla_model.py

```diff
@@ -26,6 +27,22 @@
 _DEVICE_CONTEXTS = dict()
 _DEVICE_CONTEXTS_LOCK = threading.Lock()
 
+_WORLD_SIZE = None
+_ORDINAL = None
+
+def _init_world_size_ordinal():
+  global _WORLD_SIZE, _ORDINAL
+
+  if not pjrt.using_pjrt():
+    return
+
+  # We don't support V3-8. See Note [V3-8 Threading]
+  if tpu.version() <= 3:
```

Comment:
What would this return if we are on a GPU or CPU device.

Reply:
Added another guard.